src/Spryker/Zed/RabbitMq/RabbitMqConfig.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq;
+
+use Spryker\Shared\RabbitMq\RabbitMqConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+use Spryker\Zed\RabbitMq\Business\Model\Exchange\ExchangeInfo;
+
+class RabbitMqConfig extends AbstractBundleConfig
+{
+    /**
+     * @return string
+     */
+    public function getApiExchangesUrl()
+    {
+        return sprintf('http://%s:%s/api/exchanges', 'localhost', 15672);
```

Comment:
I would like to have this in environment config

Reply:
Also, shouldnt we use either https or at least rel // by default?